### PR TITLE
Add community links for Austria

### PR DESCRIPTION
**Pull request by hikemaniac**
This PR adds community links for Austria obtained from their website, [openstreetmap.at](https://www.openstreetmap.at). It includes their OSM Mailinglist and their Twitter account. They also have a Google Plus and a Flickr account but I didn't find a type for them.

I just copied the geojson files from Belgium and inserted the correct links, country codes and coordinates. However I didn't run `npm run test` because npm was missing from my nodejs-macOS package somehow and so I'm not totally convinced everything is right. Please verify the correctness of my commits.

### CODE REVIEW

**Comment by bhousel** [comment]
Is `info@openstreetmap.at` a good email contact for questions about the community?

**Comment by hikemaniac** [comment]
I'm not sure if it is, as I'm not a heavy user of openstreetmap.at. It is mentioned as contact address from the local organisation and also seems to be the only e-mail address on their homepage. 

By the way, could we add a `type` for Google Plus? I think it's likely, that some other local communites also have a Google Plus account. Font Awesome also has an icon for Google Plus. Adding Flickr as a type is probably not really useful, as I assume not many OSM communities are part of it, even though I didn't investigate it closely. 

Unfortunately both OSM Austrias Google+ and Flickr accounts seem to have become pretty inactive. Should they still be added?

**Comment by bhousel** [comment]
> Unfortunately both OSM Austrias Google+ and Flickr accounts seem to have become pretty inactive. Should they still be added?

No - we don't want to steer users towards anything inactive.
This index is for the sort of resources that a new mapper could stop by and meet people or ask questions about OSM.

For example, If you have someone who just signed up for OSM and they made their first edit, what are the top 3 or 4 resources that you want them to see?   Maybe the openstreetmap.at local chapter, the Twitter account, and maybe the `talk-at` mailing list - they will see OSM on Reddit too because that is global.  We don't want to waste that opportunity by showing them an inactive Flickr account.

File: i18n/en.yaml [hunk 1]
```diff
@@ -68,6 +68,12 @@ en:
   OSM-TW-mailinglist:
     name: OpenStreetMap Taiwan Mailinglist
     description: Talk-tw is an Official Mailinglist for the Communities to dicuss matters about Taiwan
+  at-twitter:
+    name: OpenStreetMap Austria Twitter
+    description: Mappers and OpenStreetMap facebook in Belgium
```
Comment (tyrasd) [inline i18n/en.yaml]: this should be _Austria_ (instead of _Belgium_), right?

**Comment by hikemaniac** [comment]
Thank you @tyrasd for spotting this. I fixed it now.

**Pull request merged**